MMM-MBTA.css

```diff
@@ -26,45 +26,35 @@
   color: #5da9e8;
 }
 
-.alerts {
+.MMM-MBTA .alerts-header {
   margin-top: 6px;
 }
 
-.MMM-MBTA .alert {
-  display: inline-block;
-  position: relative;
-  width: 100%;
-  height: 100%;
-  margin: 0;
-  text-align: left;
-  white-space: nowrap;
-  /* Starting position */
-  -moz-transform:translateX(100%);
-  -webkit-transform:translateX(100%);  
-  transform:translateX(100%);
-  /* Apply animation to this element */  
-  -moz-animation: td 17s linear infinite;
-  -webkit-animation: td 17s linear infinite;
-  animation: td 17s linear infinite;
+.MMM-MBTA .alerts {
+  overflow: hidden;
+  max-width: 300px;
+  display: flex;
 }
-/* Move it (define the animation) */
-@-moz-keyframes td {
-  0%   { -moz-transform: translateX(100%); }
-  100% { -moz-transform: translateX(-100%); }
+
+.MMM-MBTA .alerts-wrapper {
+  display: flex;
+  white-space: nowrap;
 }
-@-webkit-keyframes td {
-  0%   { -webkit-transform: translateX(100%); }
-  100% { -webkit-transform: translateX(-100%); }
+
+.MMM-MBTA .alerts-wrapper.animate {
+  /* start the alert off screen */
+  padding-left: 100%;
+  transform: translateX(0%);
+  animation: alert calc(0.075s * var(--char-count, 100)) linear infinite;
```

Comment:
Set a CSS variable in JS with the number of characters the alerts are and use it in the animation timing.